ui/DesignSystem/Primitive/Button.svelte

```diff
@@ -3,54 +3,36 @@
 
   // vanilla | primary | secondary | transparent | outline
   export let variant = "primary";
-  // big | small
-  export let size = "big";
 
   export let disabled = null;
   export let icon = null;
   export let style = null;
   export let dataCy = null;
 
-  const iconClass = icon ? (size === "big" ? "icon" : "icon-small") : null;
+  const iconClass = icon ? "icon" : null;
```

Comment:
@rudolfs wondering if we can simplify this now

Reply:
I don't see a way at the moment.